src/SolutionTemplate/UnoSolutionTemplate/UnoQuickStart.Shared/Shared.vstemplate

```diff
@@ -14,6 +14,7 @@
     <EnableLocationBrowseButton>true</EnableLocationBrowseButton>
     <CreateInPlace>true</CreateInPlace>
     <Icon>__TemplateIcon.png</Icon>
+	<Hidden>true</Hidden>
```

Comment:
Indentation is off. (Not a big deal, just pointing out because there are so many other whitespace changes)